base_ubl/models/ubl.py

```diff
@@ -260,6 +260,8 @@ def _ubl_add_supplier_party(
                 ), "partner is wrong"
             else:
                 partner = company.partner_id
+        else:
```

Comment:
when does it happen?

Reply:
Basically if company is not passed to method  _ubl_add_supplier_party error occured. So I've added custom error for making company field required for this method.